src/pages/dashboard/charts/_Map.js

```diff
@@ -144,26 +128,37 @@ export default function Map({
     [maxValue],
   )
 
+  const isMobile = window.screen.availWidth < viewportSm
```

Comment:
Could be an enhancement, but it might be good to throw a window resize listener on this (with a debounce) and resent `isMobile`. Not a huge use case for laptop/desktop users, but could/should change if mobile user changes orientation.

Reply:
added #528